pyvista/core/utilities/features.py

```diff
@@ -13,8 +13,11 @@
 from .helpers import wrap
 
 
-def voxelize(mesh, density=None, check_surface=True):
-    """Voxelize mesh to UnstructuredGrid.
+def voxelize(mesh, density=None, vol=False, check_surface=True):
```

Comment:
I cannot find `pv.voxelize` in the documentation, can it be added in this PR (in addition to a second function of it is added)?

Reply:
resolved in #5133.